tests/qunit/tests/assets/dev/js/editor/container/container.spec.js

```diff
@@ -1,4 +1,4 @@
-import ElementsHelper from 'elementor-tests-qunit/core/editor/document/elements/helper';
+import ElementsHelper from 'elementor/tests/qunit/tests/assets/dev/js/editor/document/elements/helper';
```

Comment:
Remove the alias `elementor-tests-qunit` from alises list.

Reply:
Handled.